docs/syntax/imports.md

```diff
@@ -112,31 +141,19 @@ the name provided after the `as` keyword, if provided).
 
 ### Unqualified Exports
 
-Unqualified exports are broken up into three main categories:
+Unlike imports, exports cannot be used with the `all` and `hiding` keywords. So
+the only supported syntax is to export a list of names with _restricted
+exports_.
 
-1. **Unrestricted Exports:** These export all symbols from the module or type
-   into the current scope. They consist of the keyword `from`, followed by a
-   qualified module name, followed by an optional rename part (using the `as`
-   keyword), then the keywords `export all`. For example:
-   ```
-   from Standard.Base.Data.List as Builtin_List export all
-   ```
-2. **Restricted Exports:** These export a specified set of names, behaving as
-   though they were redefined in the current scope. They consist of the keyword
-   `from`, followed by a qualified module or type name (with optional
-   `as`-rename), then the word `export` followed by a coma-separated list of
-   names to be exported. For example:
-   ```
-   from Standard.Base.Data.List export Cons, Nil, from_vector
-   ```
-3. **Hiding Exports:** These are the inverse of restricted exports, and export
-   _all_ symbols other than the named ones. They consist of the `from` keyword,
-   followed by a qualified module or type name (with optional `as`-rename), then
-   the words `export all hiding`, followed by a coma-separated list of names to
-   be excluded from the export. For example:
-   ```
-   from Standard.Base.Data.List export all hiding from_vector, Nil
-   ```
+**Restricted Exports:** These export a specified set of names, behaving as
+though they were redefined in the current scope. They consist of the keyword
+`from`, followed by a qualified module or type name (with optional `as`-rename),
+then the word `export` followed by a coma-separated list of names to be
```

Comment:
```suggestion
then the word `export` followed by a comma-separated list of names to be
```

Reply:
Applied in 5b1fd5cf1e0f115bb0b637edbae81198277adbca